src/components/styles/Home.css

```diff
@@ -0,0 +1,325 @@
+  left: 0;
+  right: 0;
+  background-color: #e70983;
+  align-items: center;
+  padding: 20px 20px;
+  justify-content: space-between;
+  z-index: 5;
+}
+
+.logo {
+  display: none;
+}
+
+.goBack {
+  display: none;
+  transition: all 100ms ease;
+}
+
+.goBack:hover {
+  transform: translateX(-4px);
+  color: black;
+  font-size: 3rem;
+}
+
+.home::-webkit-scrollbar {
+  display: none;
+}
+
+.covid-19-report > *:first-child {
+  margin: 0 10px;
+  color: black;
+  font-size: 2rem;
+  transition: all 0.5s ease;
+}
+
+.covid-19-report > *:first-child:hover {
+  animation: pulse 1s infinite;
+}
+
+@keyframes pulse {
+  0%,
+  100% {
+    transform: scale(1);
+  }
+
+  50% {
+    transform: scale(1.2);
+  }
+}
+
+.navbar > *:nth-child(4):hover {
+  transition: all 0.8s ease;
+  transform: rotateZ(180deg);
+}
+
+.covid-19-report {
+  padding: 0;
+  margin: 0;
+  font-weight: 700;
+  font-size: 1.2rem;
+  color: #ffae44;
+}
+
+.searchBar {
+  align-items: center;
+  position: relative;
+}
+
+.searchBar > input {
+  border: 2px solid white;
+  background-color: rgba(0, 0, 0, 0);
+  width: 50px;
+  padding: 0.6rem 0.4rem;
+  opacity: 0;
+  border-radius: 1.2rem;
+  transition: all 0.4s ease;
+  color: white;
+}
+
+.searchBar > input::placeholder {
+  color: rgb(151, 151, 151);
+  font-family: "Poppins", sans-serif;
+  font-weight: 500;
+}
+
+.searchBar > input:focus {
+  width: 200px;
+  opacity: 1;
+}
+
+.search-icon {
+  position: absolute;
+  top: 8px;
+  right: 16px;
+  pointer-events: none;
+}
+
+.Info {
+  background-color: #e70983;
+  margin: 70px 8px 4px 8px;
+  padding: 10px 20px;
+  position: relative;
+}
+
+.Info::before {
+  content: "";
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: rgba(0, 50, 73, 0.6);
+  background-image: url(/src/images/world2.jpg);
+  background-size: contain;
+  filter: brightness(0.4);
+}
+
+.Info-Wrapper {
+  display: flex;
+  position: relative;
+  justify-content: space-between;
+  margin-top: 10px;
+  flex-wrap: wrap;
+}
+
+.global,
+.confirmed-text-home,
+.deaths-text {
+  margin: 0;
+  position: relative;
+  color: white;
+}
+
+.global {
+  font-size: 1.8rem;
+  margin: 15px 0;
+  text-align: center;
+  color: white;
+}
+
+.global i {
+  margin-right: 0.5rem;
+  font-size: 2rem;
+}
+
+.confirmed-text-home,
+.deaths-text {
+  font-weight: 700;
+  color: white;
+}
+
+.confirmed,
+.deaths {
+  color: #6f4;
+  font-weight: 600;
+  margin: 0;
+  font-size: 20px;
+}
+
+.deaths {
+  color: rgb(228, 6, 6);
+  font-size: 20px;
+}
+
+.Info > p {
+  margin: 1px;
+  padding: 0;
+}
+
+.second-row {
+  display: grid;
+  grid-template-columns: repeat(3, 250px);
+  box-sizing: border-box;
+  gap: 3px;
+  padding: 4px 8px;
+}
+
+.section {
+  display: flex;
+  justify-content: space-between;
+  background-color: #e70983;
+  padding: 10px;
+  user-select: none;
+  position: relative;
+  border-radius: 2px;
+  outline: none;
+  transition: all 0.3s ease;
+  color: white;
+}
+
```

Comment:
- [ ] Well done styling the app however I recommend alternating the colors between elements, as you can see on the caption there are two shades of color alternated between elements.

<img width="433" alt="Screenshot 2023-10-20 at 9 29 05 PM" src="https://github.com/hydaline-code/covid-19-infohub/assets/53347966/d5db6197-498f-4d2e-9e4a-a2875c8e8927">